tensorflow/core/api_def/base_api/api_def_Expm1.pbtxt

```diff
@@ -2,6 +2,19 @@ op {
   graph_op_name: "Expm1"
   summary: "Computes exponential of x - 1 element-wise."
   description: <<END
-I.e., \\(y = (\exp x) - 1\\).
+  i.e. exp(x) - 1 or e^(x) - 1, where `x` is the input tensor.
+  `e` denotes Euler's number and is approximately equal to 2.718281.
+
+  ```python
+  x = tf.constant(2.0)
+  tf.math.expm1(x) ==> 6.389056
+
+  x = tf.constant([2.0, 8.0])
+  tf.math.expm1(x) ==> array([6.389056, 2979.958], dtype=float32)
+
+  #Refer `exp` for explanation
```

Comment:
"Please refer to `exp` for an explanation."

Do you intend to add this kind of note to all functions which operate on complex numbers? Almost all functions do. I'm not sure we have to teach all our users about complex numbers. If they use them, they likely know how they work. If they don't, there's not much these examples will do to help them.

If we do want to add a uniform not and explanation about complex numbers, the docstring for exp is probably not the best place for it. I'd prefer to link out to wikipedia and keep the explanations here to a minimum.